packages/core/redux/bitcoin-like-blockchain/utils.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2017–2018, LaborX PTY
+ * Licensed under the AGPL Version 3 license.
+ */
+
+import uuid from 'uuid/v1'
+import { TxEntryModel } from '../../models'
+
+/**
+ * Create new TxEntryModel from provided data
+ * @param {Object} entry - TODO
+ * @param {Object} options - TODO
+ * @return {TxEntryModel}
+ */
+export const createBitcoinTxEntryModel = (entry, options = {}) => {
+  return new TxEntryModel({
+    key: uuid(),
+    isSubmitted: true,
+    isAccepted: false,
+    walletDerivedPath: options && options.walletDerivedPath,
```

Comment:
I know that this is copy-paste, but `options` always true here...

Reply:
Actually, no. You can check the following example:
```
const checkOptions = (options) => options && options.walletDerivedPath;
console.log(checkOptions({ walletDerivedPath: 'value '}))
console.log(checkOptions({}))
```

Reply:
At line 15 `options` has default value `{}`. 
![screen shot 2018-10-02 at 13 47 36](https://user-images.githubusercontent.com/661889/46344442-d39d1700-c649-11e8-9304-8b78e84ce57d.png)

Reply:
But I still can pass `options` as the second argument of the `createBitcoinTxEntryModel`. The default value will be ignored in this case

Reply:
Understood. `walletDerivedPath: options.walletDerivedPath` is enough